WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -184,6 +185,27 @@ - (void)willRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrie
     [self.tableViewHandler refreshCachedRowHeightsForWidth:width];
 }
 
+- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
+{
```

Comment:
According to the docs this API is 8.0 or later. Adding the declaration won't cause problems on iOS 7, but it won't be executed either.  Is this by design?